cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -4643,7 +4644,7 @@ macro(build_awssdk)
   set(AWSSDK_CMAKE_ARGS
       ${AWSSDK_COMMON_CMAKE_ARGS}
       -DOPENSSL_ROOT_DIR=${OPENSSL_ROOT_HINT}
-      -DBUILD_DEPS=OFF
+      -DBUILD_DEPS=ON # We need to build aws-crt-cpp from source
```

Comment:
Can we build `aws-crt-cpp` by ourselves instead of using this and `prefetch_crt_dependency.sh` like other dependencies such as `aws-c-common` for portability?

Reply:
I see. I'll try building it manually.